src/main/resources/io/jenkins/plugins/analysis/core/steps/IssuesRecorder/help-buildOnError.html

```diff
@@ -0,0 +1,2 @@
+By default, static analysis results are only recorded for stable or unstable builds, but not for failed builds:
```

Comment:
I'm not sure if this works if it is placed at the publish-parameters level. Maybe we need it 3 times, for `IssuesRecorder`, `PublishIssuesStep`, `RecordIssuesStep`.

Reply:
Yeah , lets add it three times and remove if not required :)